tests/modules/nf-core/purecn/coverage/main.nf

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { PURECN_COVERAGE } from '../../../../../modules/nf-core/purecn/coverage/main.nf'
+
+workflow test_purecn_coverage {
+
+    meta = [ id:'test' ]
+    input_bam = [ meta,
+                  file("https://raw.githubusercontent.com/lima1/PureCN/master/inst/extdata/ex1.bam", checkIfExists: true),
+                  file("https://raw.githubusercontent.com/lima1/PureCN/master/inst/extdata/ex1.bam.bai", checkIfExists: true)
+                  ]
+    input_intervals = ["https://raw.githubusercontent.com/lima1/PureCN/master/inst/extdata/ex1_intervals.txt"]
```

Comment:
Doesn't this work with the data already in the test datasets repository?

Reply:
Unfortunately, `scanBam` gives errors when reading from the datasets in test repository:
```
seqlevels(param) not in BAM header:
seqlevels: ‘seq1’, ‘seq2’
```
Do you think it would be simpler to continue using the datasets available in the Purecn repository, or do you have a different suggestion?

Reply:
The main problem is that we don't have control over the availability of these files. So if the developers were to remove these files, we won't be able to test anymore. A possible solution is to add these files to the repository.

Reply:
Thanks, we'll see about doing a PR there. I'll read up on the guidelines.

Reply:
https://github.com/nf-core/modules/pull/3441#event-9299968586 address the issue. Going to change the code asap

Reply:
Done.